core/src/main/java/com/klaytn/caver/wallet/KeyringContainer.java

```diff
@@ -127,11 +168,34 @@ public AbstractKeyring newKeyring(String address, String[] keys) {
     }
 
     /**
-     * Creates a role-basd type keyring instance with given parameters and add it to the keyringContainer.
-     * KeyringContainer manages Keyring instance using Map {string:Keyring} which has address as key value.
+     * Creates a role-basd type keyring instance with given parameters and add it to the keyringContainer.<p>
+     * KeyringContainer manages Keyring instance using Map {string:Keyring} which has address as key value.<p>
+     * <pre>Example :
+     * {@code
+     * String address = caver.wallet.keyring.generate().getAddress();
+     * String[][] privateKeyArr = {
```

Comment:
This also you can use `generateRoleBasedKeys`.

Reply:
I modified it.